src/Compilers/CSharp/Portable/Syntax/SyntaxFacts.cs

```diff
@@ -496,5 +498,34 @@ internal static bool IsSimpleProgramTopLevelStatement(GlobalStatementSyntax synt
         {
             return IsTopLevelStatement(syntax) && syntax.SyntaxTree.Options.Kind == SourceCodeKind.Regular;
         }
+
+        internal static bool HasAwaitOperations(SyntaxNode node)
```

Comment:
>internal [](start = 8, length = 8)

Consider using `#nullable enable` around these new extension methods so it's clear which allow `null` arguments.